app.json

```diff
@@ -6,6 +6,11 @@
       "bundler": "metro"
     },
     "name": "ke.droidcon.reactnative",
-    "slug": "ke.droidcon.reactnative"
+    "slug": "ke.droidcon.reactnative",
+    "splash": {
+      "image": "./assets/images/splash.png",
+      "resizeMode": "contain",
+      "backgroundColor": "#000000"
```

Comment:
This will always display a black background on the standalone apps even when the device is in light mode. We should use different backgrounds for light and dark:

```json
{
  "expo": {
    "userInterfaceStyle": "automatic",
    "scheme": "acme",
    "web": {
      "bundler": "metro"
    },
    "name": "ke.droidcon.reactnative",
    "slug": "ke.droidcon.reactnative",
    "ios": {
      "splash": {
        "image": "./assets/images/splash.png",
        "resizeMode": "contain",
        "backgroundColor": "#ffffff",
        "dark": {
          "image": "./assets/images/splash.png",
          "resizeMode": "contain",
          "backgroundColor": "#000000"
        }
      }
    },
    "android": {
      "splash": {
        "image": "./assets/images/splash.png",
        "resizeMode": "contain",
        "backgroundColor": "#ffffff",
        "dark": {
          "image": "./assets/images/splash.png",
          "resizeMode": "contain",
          "backgroundColor": "#000000"
        }
      }
    }
  }
}
```